lib/sender.js

```diff
@@ -51,10 +65,92 @@ class Sender extends Writable {
     if (this._options.debug) {
       console.log('%s', converted.trim());
     }
-    return this._socket.write(converted, error => this._handleError(error,
-      callback))
+    const writeRes = this._socket.write(
+      converted, error => this._handleError(error, callback));
+    return this._relp.enabled ? writeRes && this._relp.txno : writeRes;
```

Comment:
it looks like both `send` and `resend` now return either `false` or the `txno` of the sent event. Maybe this should be documented in th jsdoc of the methods. Also, I wonder if it should return `-1` instead of false in order to avoid polymorphism.

Reply:
`send` is also used under regular syslog, so I'll keep returning `true`/`false` under that scenario.

Besides that, I'm ok with the idea of returning `txno`/`-1` when sending data using RELP.

8e1f47a